schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+
+CREATE TABLE animals(
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL
 );
```

Comment:
optional:
in your table you can add ``id int NOT NULL IDENTITY(1,1)`` to make the id NOT NULL and auto increment your table

Reply:
Thanks, @alexander16108 lexander16108 for the suggestion. I will work on that.

Cheers.